github/repos.go

```diff
@@ -538,6 +540,12 @@ type RequiredPullRequestReviews struct {
 	IncludeAdmins bool `json:"include_admins"`
 }
 
+// AdminEnforcement represents the configuration to enforce required status checks for repository administrators.
+type AdminEnforcement struct {
+	URL     *string `json:"url,omitempty"`
+	Enabled bool    `json:"enabled"`
```

Comment:
What's the reason `URL` is a pointer `*string` and has `,omitempty`, unlike `Enabled`?

I think I get the reason, but it seems a bit arbitrary. I don't have any suggestions for improvement, so this is okay with me.